content/en/docs/releases/supported-releases/index.md

```diff
@@ -30,10 +30,16 @@ assistance. Separately, 3rd parties and partners may offer longer-term support s
 |Type              | Support Level                                                                                                         | Quality and Recommended Use
 |------------------|-----------------------------------------------------------------------------------------------------------------------|----------------------------
 |Development Build | No support                                                                                                            | Dangerous, may not be fully reliable. Useful to experiment with.
-|Minor Release     | Support is provided until 3 months after the next minor release                                                       | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.
+|Minor Release     | Support is provided until 4 and a half months after the next minor release                                            | Safe to deploy in production. Users are encouraged to upgrade to these releases as soon as possible.
```

Comment:
I believe it's 6 weeks after the second minor release following this minor release. As we do quarterly, it works out to a similar date, but best to say they have 6 weeks following the second minor release to get upgraded.